lib/routes/index.js

```diff
@@ -44,6 +47,29 @@ module.exports = function routing(app, options, callback) {
       res.end('ok');
     });
 
+    /**
+     * Base for swagger definitions
+     */
+    app.routes.use('/api-docs', swaggerUI.serve, swaggerUI.setup(swaggerDefs));
+
+    /**
+      * @swagger
+      * /api/packages/{pkg}:
+      *   get:
+      *     summary: Gets info about one package
+      *     produces:
+      *       - "application/json"
+      *     parameters:
+      *       - in: path
+      *         name: pkg
+      *         required: true
```

Comment:
Is it? if you don't provide it, you get all the packages right? Is that how it works?

Reply:
Either this or the route is wrong. since its marked as not required in the route.